.travis.yml

```diff
@@ -18,5 +19,5 @@ deploy:
     secure: J8ZYI+2vEbILgSalnTi1wZj374kTa+6yY67SW7uMBDKx6AQprN1Qxc00B7OmrI90xTo6Ia2wSzFPjsiPbh8BHnLI8Eoz0SoHAa5iPORx5XztLJN6kQsjeQWNuCBP5wjvDtsYlJLJsNppjun0Ywnl0w6rsFIBfcbHYRHrovntB8qiuGY5ZSq9Rnki0oDgilwIMLCUlOx1ajB+c6oRq59cNn5TKbuGh7Etb1plzsuPLDkrjrtaw9RcDp49P6jJFNCSsKNJgpy8Ki7QRVA52l16SfrRfctAO98NR6+Y+0kjUqbm+CEcAtMnzmF7/Nqi1VLfP9tRYwR0HbNh4X9yrzfOu1ilF8ZHedB0LPaAu0GdTbPJwkXVweueDQJlAYY5X8rcPbXEYUo5aEL082UhSreVFO/Q7FaG+JmbDqnMDzj2cS2zV1ruPfv8C319rSUgKRFe4GZMUcttbqZfAz490W7Y3PAyBeS7gjG8fxGyv/5iq42JYC60CZIllVLxinwRzAff3zSf8xUhCIZFjGiNb9qGaEj8jo9/vNkUfnCGFrshMEmRqUjS6ae/wS5qPlJ4pCfNCMQDNAwbDeVk5Tc9FmPRtd1lA7j63vAqe9zsKnaMzBmYAPY+wEMvOl8n+SxAylKLy+v44H8MX7nykd58qwdOW8kyYWJqAK7QrwxgnvD8n5Y=
   on:
     tags: true
-    repo: trailsjs/trailpack-express4
+    repo: trailsjs/trailpack-express
```

Comment:
@jaumard FYI, due to the way Travis computes the hashes for their secure keys (line 19), changing this line will usually cause the key to fail. I'll submit this change in a separate PR.